### ISSUE TITLE
clear Recycle Bin

### ISSUE THREAD

**Issue by Famku** [fr0tt/benotes#74]
Hi,

maybe a stupid question, but how to clear the Recycle Bin?

I cant see any option to delete the old entries, just for restore.

**Comment by fr0tt** [comment]
Hi,

currently there is no option to do that (because I didn't seem necessary to me since the amount of space allocated isn't concerningly huge).
However would you like to be able to do that ? ;)

**Comment by Famku** [comment]
yes, you are right about the disk space, but a Recycle Bin sometimes have to be cleared  ;)

I like the ability to empty the trash

**Comment by roughnecks** [comment]
I would like to see this feature as well.

**Comment by fr0tt** [comment]
Alright. 
Would you like to just have a button for it or should there be some kind of protective mechanism like for example an "are you sure?"-modal after pressing the button ?

**Comment by roughnecks** [comment]
Maybe with the warning it's better! Thanks
